README.md

```diff
@@ -1,2 +1,52 @@
-# angular-what-is-a-service-readme
-<p data-visibility='hidden'>View <a href='https://learn.co/lessons/angular-what-is-a-service-readme' title='angular-what-is-a-service-readme'>angular-what-is-a-service-readme</a> on Learn.co and start learning to code for free.</p>
+# What is a Service?
+
+## Overview
+
+We've thrown around the idea of services before - let's dig in deep and learn all about them!
+
+## Objectives
+
+- Describe the role of a Service
+- Describe dependency injection to inject Services
+- Inject a Service into a Controller
+- Use the Service inside the Controller
+
+## Our little helpers
+
+Services are helpers. They're JavaScript functions and are responsible for doing specific tasks only. For example, we might have a `UserService` that communicates with our API for everything user based. Or a `NotificationService` to deal with showing notifications to the user.
```

Comment:
Really nice example here. This makes super clear the types of responsibilities our services will have.